client/lib/text-utils/index.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * External Dependencies
+ */
+import JsDiff from 'diff';
```

Comment:
Can we `import { diffWords } from 'diff';` here instead of accessing property below?